### PR TITLE
Merge stable to master

### PR DESCRIPTION
**Pull request by geoffnichols**

### CODE REVIEW

File: configs/components/libxml2.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 component "libxml2" do |pkg, settings, platform|
-  pkg.version "2.9.3"
-  pkg.md5sum "daece17e045f1c107610e137ab50c179"
-  pkg.url "http://xmlsoft.org/sources/libxml2-#{pkg.get_version}.tar.gz"
+  pkg.version "2.9.4"
+  pkg.md5sum "ae249165c173b1ff386ee8ad676815f5"
+  pkg.url "http://buildsources.delivery.puppetlabs.net/#{pkg.get_name}-#{pkg.get_version}.tar.gz"
```
Comment (shrug) [inline configs/components/libxml2.rb]: I think we want to be using the upstream urls

File: configs/components/libxslt.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 component "libxslt" do |pkg, settings, platform|
-  pkg.version "1.1.28"
-  pkg.md5sum "9667bf6f9310b957254fdcf6596600b7"
-  pkg.url "http://xmlsoft.org/sources/libxslt-#{pkg.get_version}.tar.gz"
+  pkg.version "1.1.29"
+  pkg.md5sum "a129d3c44c022de3b9dcf6d6f288d72e"
+  pkg.url "http://buildsources.delivery.puppetlabs.net/#{pkg.get_name}-#{pkg.get_version}.tar.gz"
```
Comment (shrug) [inline configs/components/libxslt.rb]: I think we want to be using the upstream urls

File: configs/components/rubygem-mini_portile2.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 component "rubygem-mini_portile2" do |pkg, settings, platform|
-  pkg.version "2.0.0"
-  pkg.md5sum "e608463ac8081fe600f7bb6ea46c3e64"
-  pkg.url "https://rubygems.org/downloads/mini_portile2-#{pkg.get_version}.gem"
+  pkg.version "2.1.0"
+  pkg.md5sum "d771975a58cef82daa6b0ee03522293f"
+  pkg.url "http://buildsources.delivery.puppetlabs.net/mini_portile2-#{pkg.get_version}.gem"
```
Comment (shrug) [inline configs/components/rubygem-mini_portile2.rb]: I think we want to be using the upstream urls

File: configs/components/rubygem-pkg-config.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
+component "rubygem-pkg-config" do |pkg, settings, platform|
+  pkg.version "1.1.7"
+  pkg.md5sum "2767d4620b32f2a4ccddc18c353e5385"
+  pkg.url "http://buildsources.delivery.puppetlabs.net/pkg-config-#{pkg.get_version}.gem"
```
Comment (shrug) [inline configs/components/rubygem-pkg-config.rb]: We should probably point this at an upstream url

File: configs/components/rubygem-mini_portile2.rb (hunk 3 above)
Comment (geoffnichols) [inline configs/components/rubygem-mini_portile2.rb]: Thanks, I totally missed that. I'll update.

**Pull request closed without merging**